imports/api/partners/partners.js

```diff
@@ -169,7 +169,7 @@ Factory.define('supplier', Partners, {
   contact: {
     address: () => faker.address.streetAddress('###'),
     phone: () => faker.phone.phoneNumberFormat(1),
-    email: () => `${faker.random.word()}.demouser@honline.hu`,
+    email: () => `${faker.name.firstName()}.demouser@honline.hu`,
```

Comment:
Itt nem lehet az gond, ha veletlenul 2 kulonbozonek sorsoja azt hogy Daniel?
Mert akkor letre akarhat hozni 2 user-t ua email cimmel. Ami azt hiszem not allowed.